illumos-utils/src/zfs.rs

```diff
@@ -82,7 +79,7 @@ enum EnsureFilesystemErrorRaw {
     MountOverlayFsFailed(crate::ExecutionError),
 }
 
-/// Error returned by [`Zfs::ensure_filesystem`].
+/// Error returned by [`Zfs::ensure_dataset`].
 #[derive(thiserror::Error, Debug)]
 #[error("Failed to ensure filesystem '{name}': {err}")]
 pub struct EnsureFilesystemError {
```

Comment:
Do you want to rename this type to match the method name change? (I don't have strong feelings either way; the name of error enums is not high on my personal list of "code clarity important things".)

Reply:
Renamed!